doc/api/cli.md

```diff
@@ -1969,6 +1969,9 @@ A regular expression that configures the test runner to only execute tests
 whose name matches the provided pattern. See the documentation on
 [filtering tests by name][] for more details.
 
+If both the `--test-name-pattern` and `--test-skip-pattern` arguments are
```

Comment:
I still think this is vague. This could be interpreted as "if you pass both flags, `--test-skip-pattern` is ignored globall." That's why I recommended text that was specific to individual tests.

Reply:
Well, it's not wrong. If `--test-name-pattern` is supplied, `--test-skip-pattern` *is* ignored globally. In the checking algorithm, it checks the patterns like:
```js
    if (testNamePatterns) {
      return !testMatchesPattern(this, testNamePatterns);
    }

    if (testSkipPatterns) {
      return testMatchesPattern(this, testSkipPatterns);
    }
```

Reply:
I could change it to, but I'd need to test to see if it'd work differently:
```js
return !testMatchesPattern(this, testNamePatterns) && testMatchesPattern(this. testSkipPatterns);
```

Reply:
Fair enough. I'm OK with this text.

Reply:
Don't worry, I changed the logic a bit, and now, if they are both supplied, any tests need to match **both** arguments. I think that makes more sense.